app/settings.py

```diff
@@ -126,7 +126,7 @@ def utcoffset_or_fail(date_value, key):
 EQ_SESSION_ID = "eq-session-id"
 
 EQ_LIST_ITEM_ID_LENGTH = 6
-MAX_NUMBER = 9999999999
+MAX_NUMBER = 9_999_999_999_999
```

Comment:
This is only max 13 digits. Card say max 15 for positive, 14 for negative. The decimal shouldn't matter?

Do we not enforce a MIN?

Reply:
I counted the 2 decimal places as part of the 15/14. The card says 15/14 **digits**, and it seems online consensus is digits == integer for cases like this i.e. decimals don't count, so I agree now it should be 15/14 whole numbers and decimals are don't count towards that. 
Revoking approval 👍

Reply:
👍 Ya, I think decimal should only matter if we were dealing with finite memory/characters.

@Yuyuutsu Check with Donna, what downstream can handle, should we be limiting the number of characters, or the total value to 15 integers. Ask if 15 digits with decimal is supported downstream. We currently support up to 6 decimals.

Reply:
Oh okay. It was my assumption, since the card said we already support 12 digits with only 10 9's in the max_number variable. I shall message Donna. 
I was going to add Min_number variable but in some functions we default it to 0. Which certain schemas rely on, I think like stocks schemas. They don't have a minimum set in the json.

Reply:
Spoke to donna, she says that she believes it should be total value is 15 integers. Also down stream is making sure they can acoomdate 👍